layouts/partials/head/head.html

```diff
@@ -8,7 +8,11 @@
 {{- $title := partialCached "data/title" . .RelPermalink -}}
 <title>{{ $title }}</title>
 
-<link rel='canonical' href='{{ .Permalink }}'>
+{{ if .Params.canonicalurl }}
+<link rel="canonical" href="{{ .Params.canonicalurl }}">
```

Comment:
What if we use just `canonical` for the field name? I think it would be simpler. What do you think?

Reply:
i think its normally called canonical url, in medium and other platforms